upgrading/upgrade-runnning-applications/2_new_feature.md

```diff
@@ -0,0 +1,88 @@
+In this step we add a new feature to the `create-daml-app` code. Change directory to the
+`create-daml-app` project
+
+```
+cd ../create-daml-app
+```{{execute T2}}
+
+and open `daml/User.daml`{{open}} in the IDE tab and wait for it to load. We'll extend the `User`
+data with a `nickname` field, such that the beginning of the template looks like
+
+<pre class="file" data-target="clipboard">
+template User
+  with
+    username: Party
+    following: [Party]
+    nickname: Optional Text
+  where
+    ...
```

Comment:
```suggestion
```

If you copy it like this id adds the three dots and the IDE complains that it's an error.

Reply:
removed it.